src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -167,7 +167,11 @@ export class CellExecution implements IDisposable {
                 // If the cell is deleted, then dispose the request object.
                 // No point keeping it alive, just chewing resources.
                 if (e === this.cell.document) {
-                    this.request?.dispose(); // NOSONAR
+                    try {
+                        this.request?.dispose(); // NOSONAR
+                    } catch (e) {
+                        traceError(`Error during cell execution dispose: ${e}`);
```

Comment:
I think something in the new kernel furture wrapper is throwing errors, I've been running into those a couple of times as well. (basically getting errors about request being disposed).
Looks like something isn't being handled correctly.